h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderModel.java

```diff
@@ -70,20 +76,20 @@ protected boolean defaultDropConsCols() {
   }
 
   public static class TargetEncoderOutput extends Model.Output {
+
+    public final TargetEncoderParameters _parms;
+    public final int _nclasses;
+    public final IcedHashMap<String, Frame> _target_encoding_map;
+    public final IcedHashMap<String, Boolean> _te_column_to_hasNAs;
     
-    public IcedHashMap<String, Frame> _target_encoding_map;
-    public TargetEncoderParameters _parms;
-    public IcedHashMap<String, Boolean> _te_column_to_hasNAs;
-    public double _prior_mean;
```

Comment:
Completely removed `prior_mean` from output: this is unnecessary as the priorMean can easily be computed from the encodings frames, and besides, it is problematic for multiple reasons:
- meaningless in case of multiclass (there is a prior for each target value).
- the global prior should be used wisely as it leaks data, so we should not encourage anyone using it directly.